types/chai-jquery/index.d.ts

```diff
@@ -27,6 +27,10 @@ declare namespace Chai {
         disabled(): Assertion;
         (selector: string): Assertion;
     }
+
```

Comment:
According to your suggestion, here should be...
```
interface Match {
    (regexp: RegExp, message?: string): Assertion;
    (selector: string): Assertion;
}
```
ps. I commented here to make sure the bot doesn't think that we don't have conclusion yet (it still have "Awaiting reviewer.." label on the PR).